account/src/main/java/com/caldevsupplychain/account/config/CorsConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.caldevsupplychain.account.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+
+@Configuration
+public class CorsConfig {
+	@Bean
+	public WebMvcConfigurer corsConfigurer() {
+		return new WebMvcConfigurerAdapter() {
+			@Override
+			public void addCorsMappings(CorsRegistry registry) {
+				registry.addMapping("/api/account/v1/**").allowedOrigins("http://localhost:3000");
```

Comment:
It's better to move a url to a property file;
```
ublic class CorsConfig {
       @Value("${cors.url}")
       private String corsUrl;

 	@Bean
 	public WebMvcConfigurer corsConfigurer() {
 		return new WebMvcConfigurerAdapter() {
 			@Override
 			public void addCorsMappings(CorsRegistry registry) {
 				registry.addMapping("/api/account/v1/**").allowedOrigins(corsUrl);
        }
```
and `cors.url` should be define in application.yaml

Reply:
fixed